pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -28,6 +28,13 @@ const (
 	MaxBaseOsCount       = 2
 	BaseOsImageCount     = 1
 	rebootConfigFilename = types.PersistStatusDir + "/rebootConfig"
+
+	// interface name length limit as imposed by Linux kernel.
+	ifNameMaxLength = 15
+
+	// range of valid VLAN IDs
+	minVlanID = 1
+	maxVlanID = 4094
```

Comment:
Isn't it 4093?

Reply:
According to the IEEE 802.1Q specification, only 0 and 4095 are reserved and cannot be used.
4094 has no special purpose (maybe on some devices but not universally).